src/gql/queries/get-projects.ts

```diff
@@ -1,7 +1,7 @@
 import gql from "graphql-tag";
 
 export const GET_PROJECTS = gql`
-  {
+  query Projects {
```

Comment:
Every query should have a custom name. Also im not sure if theres a naming convention for queries. There seems to be a mix of both lower cased and uppercased query names.

Reply:
let's use CamelCase

Reply:
I don't have an opinion on this, but just double checking - do we want camelCase or TitleCase?